upgrade_downgrade/common.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+die() {
```

Comment:
i feel like this file already exists somewhere. could we just include it if so? @carolynhu

Reply:
Yes. It was in `test_upgrade_downgrade.sh`. I moved it to `common.sh` as I found that to be generic enough.